package.json

```
@@ -10,7 +10,8 @@
         "lint-report": "eslint --ext .ts,.tsx --format json-relative --output-file reports/eslint.json src/",
         "format": "prettier . --write",
         "preview": "vite preview",
-        "prepare": "husky install"
+        "prepare": "husky install",
+        "simDev": "concurrently \"json-server --watch db.json\" \"vite\""
```

I would maybe do it like this, because then we can also start json-db separately if we wanted to.
```suggestion
        "prepare": "husky install",
        "json-db": "json-server --watch db.json",
        "dev-db": "concurrently npm:json-db npm:dev"
```

---

I added your suggestion to the package.json